src/presets/recoil.ts

```diff
@@ -2,16 +2,45 @@ import type { ImportsMap } from '../types'
 
 export default <ImportsMap>{
   // https://recoiljs.org/docs/api-reference/core/atom/
+  // recoil v0.7.7
   recoil: [
+    'DefaultValue',
+    'RecoilEnv',
+    'RecoilLoadable',
+    'RecoilRoot',
     'atom',
+    'atomFamily',
+    'constSelector',
+    'errorSelector',
+    'isRecoilValue',
+    'noWait',
+    'readOnlySelector',
+    'retentionZone',
     'selector',
+    'selectorFamily',
+    'snapshot_UNSTABLE',
```

Comment:
I don't think we should support **every imports**, but only the high-playing ones. `DefaultValue` doesn't make sense and seems to be a type export, `*_UNSTABLE` definitely no.